src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/RenamerProjectTreeActionHandler.cs

```diff
@@ -61,6 +63,68 @@ internal partial class RenamerProjectTreeActionHandler : ProjectTreeActionHandle
             _operationProgressService = operationProgressService;
         }
 
+        public override async Task CopyAsync(IProjectTreeActionHandlerContext context, string filename, string folderToMove)
```

Comment:
Is this is folder the file is moving to?

Reply:
I will change variable names. It is confusing. It says 'copy' but it represents a file 'move'. A file is moved to the new folder 
'folderToMove'